stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```diff
@@ -29,13 +29,13 @@ Next, update the bitcoin configuration:
 
 ```toml
 server=1
+daemon=1
```

Comment:
This is only needed if running in background. docs are written for systemd, not command line invocation. 
```suggestion
```

Reply:
without daemon=1 the bitcoind will load in the terminal foreground

Reply:
daemon=1 drops bitcoind into the terminal as a background process and continues to run as a system process in the background

Reply:
> This is only needed if running in background. docs are written for systemd, not command line invocation.

either way running directly or as a systemd process daemon=1 doesnt prevent anyone from using it as a systemd process...

if anything its helpful for people that run it directly instead of with systemd because it will work either way

Reply:
then why make the change? what does it provide vs systemd which backgrounds it anyways? it's not needed